.github/CODEOWNERS

```diff
@@ -9,3 +9,5 @@
 /pubsub/            @googleapis/api-pubsub
 /spanner/           @skuruppu
 /storage/           @tritone
+/errorreporting/    @googleapis/api-logging @nicoleczhu
```

Comment:
nit: remove nicoleczhu from both as she is included in the group.

Reply:
done